### PR TITLE
duc: add page

### PR DESCRIPTION
**Pull request by AngelODeath**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/duc.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# duc
+
+> Duc: index, query and graph disk usage
```
Comment (CleanMachine1) [inline pages/common/duc.md:3]: Could you maybe think of a better description than this, because its a little vague.

File: pages/common/duc.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> More information: <https://duc.zevv.nl/>
+
+- Index the /usr directory, writing to the default database location ~/.duc.db:
+
+`duc index /usr`
```
Comment (CleanMachine1) [inline pages/common/duc.md:8]:
```suggestion
`duc index {{/usr}}`
```

File: pages/common/duc.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- List all files and directories under /usr/local, showing relative file sizes in a graph:
+
+`duc ls -Fg /usr/local`
```
Comment (CleanMachine1) [inline pages/common/duc.md:12]:
```suggestion
`duc ls -Fg {{/usr/local}}`
```

File: pages/common/duc.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+List all files and directories under /usr/local using tree view:
+
+`duc ls -Fg -R /usr/local`
```
Comment (CleanMachine1) [inline pages/common/duc.md:16]:
```suggestion
`duc ls -Fg -R {{/usr/loca}}l`
```

File: pages/common/duc.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+Start the graphical interface to explore the file system using sunburst graphs:
+
+`duc gui /usr`
```
Comment (CleanMachine1) [inline pages/common/duc.md:20]:
```suggestion
`duc gui {{/usr}}`
```

File: pages/common/duc.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+Run the ncurses console interface to explore the file system:
+
+`duc ui /usr`
```
Comment (CleanMachine1) [inline pages/common/duc.md:24]:
```suggestion
`duc ui {{/usr}}`
```

File: pages/common/duc.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# duc
+
+> Duc: index, query and graph disk usage
+> More information: <https://duc.zevv.nl/>
+
+- Index the /usr directory, writing to the default database location ~/.duc.db:
+
+`duc index /usr`
+
+- List all files and directories under /usr/local, showing relative file sizes in a graph:
```
Comment (EmilySeville7cfg) [inline pages/common/duc.md:10]:
```suggestion
- List all files and directories under /usr/local, showing relative file sizes in a [g]raph:
```

File: pages/common/duc.md (hunk 4 above)
Comment (EmilySeville7cfg) [reply pages/common/duc.md:16]: Small typo fix:
 
```suggestion
`duc ls -Fg -R {{/usr/local}}`
```

😄

File: pages/common/duc.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+`duc ls -Fg /usr/local`
+
+List all files and directories under /usr/local using tree view:
```
Comment (EmilySeville7cfg) [inline pages/common/duc.md:14]:
```suggestion
List all files and directories under /usr/local using treeview recursively:
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/duc.md (hunk 1 above)
Comment (AngelODeath) [reply pages/common/duc.md]: Done

File: pages/common/duc.md (hunk 2 above)
Comment (AngelODeath) [reply pages/common/duc.md]: Done

File: pages/common/duc.md (hunk 4 above)
Comment (AngelODeath) [reply pages/common/duc.md]: Done

File: pages/common/duc.md (hunk 6 above)
Comment (AngelODeath) [reply pages/common/duc.md]: Done

File: pages/common/duc.md (hunk 8 above)
Comment (AngelODeath) [reply pages/common/duc.md]: Done

File: pages/common/duc.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# duc: Dude, where are my bytes
```
Comment (CleanMachine1) [inline pages/common/duc.md:1]:
```suggestion
# duc
```

File: pages/common/duc.md [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+> Duc is a collection of tools for indexing, inspecting and visualizing disk usage. Duc maintains a database of accumulated sizes of directories of the file system, and allows you to query this database with some tools, or create fancy graphs showing you where your bytes are.
```
Comment (CleanMachine1) [inline pages/common/duc.md:3]:
```suggestion
> Duc is a collection of tools for indexing, inspecting and visualizing disk usage. Duc maintains a database of accumulated sizes of directories of the file system, allowing queries this database, or create fancy graphs to show where data is 
```

File: pages/common/duc.md [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+> Duc is a collection of tools for indexing, inspecting and visualizing disk usage. Duc maintains a database of accumulated sizes of directories of the file system, and allows you to query this database with some tools, or create fancy graphs showing you where your bytes are.  
+> More information: <https://duc.zevv.nl/>
+
+- Index the /usr directory, writing to the default database location ~/.duc.db:
+
+`duc index {{/usr}}`
+
+- List all files and directories under /usr/local, showing relative file sizes in a [g]raph:
+
+`duc ls -Fg {{/usr/local}}`
```
Comment (EmilySeville7cfg) [inline pages/common/duc.md:12]:
```suggestion
- List all files and directories under /usr/local, showing relative file sizes in a graph:

`duc ls --classify --graph {{/usr/local}}`
```

File: pages/common/duc.md [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
+
+List all files and directories under /usr/local using treeview recursively:
+
+`duc ls -Fg -R {{/usr/local}}`
```
Comment (EmilySeville7cfg) [inline pages/common/duc.md:16]:
```suggestion
`duc ls --classify --graph --recursive {{/usr/local}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Comment by EmilySeville7cfg** [comment]
@AngelODeath, if some of long options used frequently please replace with short ones (in `common/`) but duplicate this page with just long options to `linux/`.

**Comment by AngelODeath** [comment]
Changed as requested. I hope i did it right...

File: pages/common/duc.md [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
+
+Start the graphical interface to explore the file system using sunburst graphs:
+
+`duc gui {{/usr}}`
+
+Run the ncurses console interface to explore the file system:
```
Comment (EmilySeville7cfg) [inline pages/common/duc.md:22]:
```suggestion
- Run the ncurses console interface to explore the file system:
```

**Comment by EmilySeville7cfg** [comment]
I am going to commit syntax error fixes.

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by AngelODeath — COMMENTED** [review]
Looks nice

**Review by AngelODeath — COMMENTED** [review]
I meant to review these changes, not the my own old changes

**Review by blueskyson — APPROVED** [review]
Look good to me.

**Pull request merged**